### PR TITLE
Improve testing

### PR DESCRIPTION
**Pull request by jgpruitt**
- `make test` now works whether you are inside or outside the docker container
- tests now better handle errors that cause the `actual` psql variable to be unset
- fail if no tests were run

### CODE REVIEW

File: test.sh
Comment (adolsalamanca) [inline test.sh:1]: nit, missing of file line

**Review by adolsalamanca — APPROVED** [review]
Left a tiny nit

**Review by avthars — APPROVED** [review]
LGTM

File: test.sh
Comment (jgpruitt) [reply test.sh:1]: fixed

**Pull request merged**